src/vtarget-out/inv-syn/inv_syn_cegar.cc

```diff
@@ -0,0 +1,962 @@
+/// \file Source for invariant synthesis --- using CEGAR loop
+// Hongce Zhang
+
+#include <ilang/util/log.h>
+#include <ilang/util/fs.h>
+#include <ilang/util/str_util.h>
+#include <ilang/util/container_shortcut.h>
+#include <ilang/verilog-in/verilog_analysis.h>
+#include <ilang/vtarget-out/vtarget_gen_impl.h>
+#include <ilang/vtarget-out/inv-syn/inv_syn_cegar.h>
+
+#include <memory>
+
+namespace ilang {
+
+static std::string select_script_to_run(const std::vector<std::string> & scripts, const std::string sel) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'sel' should be passed by const reference.](https://app.codacy.com/app/Bo-Yuan-Huang/ILAng/pullRequest?prid=4555809)